.github/CONTRIBUTOR_GUIDE/ct/AppName.md

```diff
@@ -0,0 +1,231 @@
+# **AppName<span></span>.sh Scripts**
```

Comment:
This script seem to be a "I am explaining each line in this install script". If this is the purpose, I may suggest an alternative approach to this:

Copying the whole script in this markdown file and using bash comments to add the relevant documentation context.

The current structure is hard to follow as the reader is not having the script in front of him/her. It is broken down in chapters. While commenting https://github.com/community-scripts/ProxmoxVE/blob/main/ct/snipeit.sh in full length, the reader is having the context visible.

E.g.

```
[...]
APP="SnipeIT"
var_tags="assat-management;foss"
var_cpu="2"
var_ram="2048"
var_disk="4"
# The operating system. Possible values are alpine, debian, ubuntu
var_os="debian"
# ...
var_version="12"
var_unprivileged="1"
[...]
```

Reply:
Put sometimes you only need a short reminder of how to do handle the credentials, or how to do handle config files.  If there is one finished script with comments, this would always be copied and things get left in there for no reason. This should not be a step by step guide for a script, more a how we do things here type of documentation. We could not ever explane any case in one finished script.